pkg/pii/pii.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2019 Tetrate
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pii
+
+import (
+	"crypto/sha256"
+	"fmt"
+
+	"bitbucket.org/creachadair/shell"
+
+	l "github.com/tetratelabs/log"
+)
+
+var logger = l.RegisterScope("pkg/pii", "filters log lines for PII data", 0)
+
+var defaultFilter, _ = Default()
```

Comment:
nit: make this a `var ( ... )` declaration:
```go
var (
	defaultFilter, _ = Default()

	istioFormat = `[%START_TIME%] "%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL%"` +
		` %RESPONSE_CODE% %RESPONSE_FLAGS% "%DYNAMIC_METADATA(istio.mixer:status)%" "%REQ(USER-AGENT)%"`

	defaultPII = map[string]bool{ ... }
)
```

Reply:
(But leave the logger declaration out - use the block to group related variables, essentially)